### DISCUSSION TITLE
P

### DISCUSSION THREAD

**Discussion by BrokNsoul21** [community/community#33887] [GitHub Education]
Hi, Selamat pagi. Bisa membantu saya?

**Comment by samus-aran** [comment]
We are happy you are here! :wave: You are more likely to get a useful response if you are explicit about what your query entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by ettaboyle** [comment]
Thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
